tests/components/sonos/test_media_player.py

```diff
@@ -170,3 +171,23 @@ async def test_play_media_music_library_playlist_dne(
     assert soco_mock.play_uri.call_count == 0
     assert media_content_id in caplog.text
     assert "playlist" in caplog.text
+
+
+async def test_partymode(
+    hass: HomeAssistant,
+    soco_factory: SoCoMockFactory,
+    async_autosetup_sonos,
+) -> None:
+    """Test partymode service call ."""
+    soco_mock = soco_factory.mock_list.get("192.168.42.2")
+
+    await hass.services.async_call(
+        MP_DOMAIN,
+        SERVICE_PARTYMODE,
+        {
+            "entity_id": "media_player.zone_a",
+        },
+        blocking=True,
+    )
+    assert soco_mock.partymode.call_count == 1
+
```

Comment:
Here i think you have a formatting problem.  There should be one blank line at the end, are there?